File: sandbox/ruby/src/TemplateGetDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

begin
    response = Dropbox::Sign::TemplateApi.new.template_get(
        "f57db65d3f933b5316d398057a36176831451a35",
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Template#template_get: #{e}"
end

File: sandbox/node/src/TemplateGetDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TemplateApi();

const templateId = "f57db65d3f933b5316d398057a36176831451a35";

apiCaller.templateGet(
    templateId,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Template#templateGet:");
  console.log(error.body);
});
